sdk/python/kfp/dsl/pipeline_channel.py

```diff
@@ -83,6 +83,10 @@ def __init__(
             task_name: Optional; The name of the task that produces the pipeline
                 channel. If provided, the task name will be sanitized to be k8s
                 compatible.
+            is_artifact_list: Optional; True if `channel_type` represents a list
+                of the artifact type. Only applies when `channel_type` is an artifact.
+            value: The actual value of the pipeline channel. If provided, the
+                pipeline channel is "resolved" immediately.
```

Comment:
Docstrings changes also need to be removed

Reply:
Done.